app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -56,12 +56,14 @@ def self.enabled?
     end
 
     def self.step_info
+      #puts "*** yo I got called"
       Idv::StepInfo.new(
         key: :how_to_verify,
         controller: self,
         next_steps: [:hybrid_handoff, :document_capture],
         preconditions: ->(idv_session:, user:) do
-          self.enabled? && idv_session.idv_consent_given
+          # TODO: We need idv_session here, not in enabled, because it's not available. :-\
+          self.enabled? && idv_session.idv_consent_given && idv_session.service_provider&.in_person_proofing_enabled
```

Comment:
Logically this check makes more sense in the `enabled?` method above, but it doesn't have `idv_session` available. That gets passed into the block here in the IdvStepConcern.